apps/form-builder/src/components/FormSubmissions/FormSubmissionTable/index.tsx

```diff
@@ -0,0 +1,90 @@
+'use client'; //this component runs with the client and thus requires 'use client'
+import * as React from 'react';
+import { GridValidRowModel } from '@mui/x-data-grid';
+import { DataGrid, GridColDef } from '@mui/x-data-grid';
+import { FormSubmissionResponse } from '@hack4impact-utk/internal-models';
+interface Props {
+  formSubmissions: FormSubmissionResponse[];
+}
+
+//FormSubmissionTable Component that takes in FormSubmissionResponse[]
+export default function FormSubmissionTable(props: Props) {
+  //create the rows for table using the map function that iterates each element in formSubmission.
+  const rows: GridValidRowModel[] = props.formSubmissions
+    .map((submission_element) => {
+      let createdAtDate;
+      let responderEmail;
+      //check if the responderEmail is a string and if it exists. If string doesn't exists, print out Anonymous.
+      if (typeof submission_element.responderEmail === 'string') {
+        if (!submission_element.createdAt) {
+          responderEmail = 'Anonymous';
+        } else {
+          responderEmail = submission_element.responderEmail;
+        }
+      } else {
+        //if not a string, return null
+        return null;
+      }
```

Comment:
For this section, the logic should be if there's not a valid email then set responderEmail to 'Anonymous'. We can do this with this if statement: `if (!submission_element.responderEmail)`.  This also accounts for empty strings.